.travis.yml

```diff
@@ -28,17 +28,20 @@ matrix:
     env: COVERALLS=true
 
 before_install:
+    - git clone --depth 1 https://github.com/Ensembl/ensembl-git-tools.git
+    - export PATH=$PWD/ensembl-git-tools/bin:$PATH
     - echo "TRAVIS_BRANCH=$TRAVIS_BRANCH"
     - export ENSEMBL_BRANCH='master'
-    - if [[ $TRAVIS_BRANCH =~ ^release\/[0-9]+$ ]]; then export ENSEMBL_BRANCH=$TRAVIS_BRANCH; fi
+    - export SECONDARY_BRANCH='main'
+    - if [[ $TRAVIS_BRANCH =~ ^release\/[0-9]+$ ]]; then export ENSEMBL_BRANCH=$TRAVIS_BRANCH; export SECONDARY_BRANCH=$TRAVIS_BRANCH; fi
     - echo "ENSEMBL_BRANCH=$ENSEMBL_BRANCH"
-    - git clone --branch $ENSEMBL_BRANCH --depth 1 https://github.com/Ensembl/ensembl-test.git
-    - git clone --branch $ENSEMBL_BRANCH --depth 1 https://github.com/Ensembl/ensembl.git
-    - git clone --branch $ENSEMBL_BRANCH --depth 1 https://github.com/Ensembl/ensembl-compara.git
-    - git clone --branch $ENSEMBL_BRANCH --depth 1 https://github.com/Ensembl/ensembl-variation.git
-    - git clone --branch $ENSEMBL_BRANCH --depth 1 https://github.com/Ensembl/ensembl-vep.git
-    - git clone --branch $ENSEMBL_BRANCH --depth 1 https://github.com/Ensembl/ensembl-funcgen.git
-    - git clone --branch $ENSEMBL_BRANCH --depth 1 https://github.com/Ensembl/ensembl-io.git
+    - git-ensembl --clone --depth 1 --branch $ENSEMBL_BRANCH ensembl-test
+    - git-ensembl --clone --depth 1 --branch $ENSEMBL_BRANCH ensembl
+    - git-ensembl --clone --depth 1 --branch $ENSEMBL_BRANCH ensembl-compara
```

Comment:
I think this should be `--branch $SECONDARY_BRANCH`

Reply:
1st comment (self addressed): never open PRs late at night! Especially, when getting older.

2nd comment: thanks @ens-bwalts  ... but the problem here is that the secondary_branch is entirely missing.
Shall update it soon.